### ISSUE TITLE
Ocean shaders not working on Linux and Mac OS X (KSP 1.2)

### ISSUE THREAD

**Issue by leinardi** [LGhassen/Scatterer#19]
There seems to be an issue with the ocean shaders, the surface of the ocean is completely transparent (the rover in the 2nd picture is floating, not flying):

![20161016093609_1](https://cloud.githubusercontent.com/assets/273338/19416038/858ca9fc-9385-11e6-96b5-c3aa9407a384.jpg)
![20161016093908_1](https://cloud.githubusercontent.com/assets/273338/19416036/858aeb08-9385-11e6-82d5-0bcc331ab06f.jpg)
![20161016093922_1](https://cloud.githubusercontent.com/assets/273338/19416035/8588fb68-9385-11e6-9e94-e7e917501912.jpg)
![20161016093953_1](https://cloud.githubusercontent.com/assets/273338/19416037/858cd4d6-9385-11e6-8a9a-92f1f1bed6d4.jpg)

In the last screenshot is visible a console error that is getting printed several time per second.

I'm running KSP 1.2 (build id 01586) and the only mods installed are Scatterer (v0.0255forKSP1.2) and Module Manager (2.7.2).

System info:
```
System:    Host: x87-deluxe Kernel: 4.4.0-43-generic x86_64 (64 bit)
           Desktop: Unity 7.4.0  Distro: Ubuntu 16.04 xenial
Machine:   System: ASUS product: All Series
           Mobo: ASUSTeK model: Z87-DELUXE v: Rev 1.xx
           Bios: American Megatrends v: 2103 date: 08/15/2014
CPU:       Quad core Intel Core i7-4770K (-HT-MCP-) cache: 8192 KB 
           clock speeds: max: 3900 MHz 1: 3500 MHz 2: 3477 MHz 3: 3500 MHz
           4: 3506 MHz 5: 3511 MHz 6: 3316 MHz 7: 3505 MHz 8: 3505 MHz
Graphics:  Card: NVIDIA GM204 [GeForce GTX 980]
           Display Server: X.Org 1.18.4 driver: nvidia
           Resolution: 2560x1440@59.95hz
           GLX Renderer: GeForce GTX 980/PCIe/SSE2
           GLX Version: 4.5.0 NVIDIA 364.19
```

**Comment by gappleto97** [comment]
+1

I'm getting the same issues on my machine.

**Comment by GiantGiraffe** [comment]
Same issue happening for me.  Mac OS El Capitan.  Machine stats below:

5k Retina iMac (Late 2014 model)
Processor:  Intel 4GHz Core i7
Memory:  32GB 1600 MHz DDR3
Graphics:  AMD Radeon R9 M295x 4096 MB (Native resolution 5120 x 2880; running KSP at 2880 x 1620)
OS:  Mac OS El Capitan 10.11.6
KSP Version:  1.2.0.1586 (1.2 release)

I'm not a developer so let me know if any further info would be useful.

**Comment by LGhassen** [comment]
This should be fixed on newest version (uploaded yesterday)

**Comment by leinardi** [comment]
I can confirm that it is solved with version 0.0256.
